src/aat/resources/features/F-1021 - On Behalf Of User Id after Case created Get events by case ID/S-1021.1.td.json

```diff
@@ -0,0 +1,51 @@
+{
+  "title": "must return list of audit events successfully for a case",
+  "_guid_": "S-1021.1",
+  "_extends_": "F-1021_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "uses the case-reference of the case just created",
+    "contains a token created as in S-597-Prerequisite",
+    "contains the Id of the case just created",
+    "Retrieve audit events by case ID",
+    "contains all audit event details under the case"
+  ],
+  "request": {
+    "pathVariables": {
+      "cid": "${[scenarioContext][childContexts][F-1021_Standard_Full_Case_Creation_Data][testData][actualResponse][body][id]}"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_200_Response",
+    "body": {
+      "auditEvents": [
+        {
+          "id" : "CREATE",
+          "summary" : "",
+          "description" : "",
+          "user_id" : "[[ANYTHING_PRESENT]]",
```

Comment:
Should this reference a particular user rather than just any?

Reply:
we don't know idam id of the user, that's the reason we are using ANYTHING_PRESENT